### DISCUSSION TITLE
<Your question or feedback about the GitHub Discussions product> GitHub will remove the "Block User" feature

### DISCUSSION THREAD

**Discussion by XuanChen1234** [community/community#121199] [Discussions] [answered]
### Select Topic Area

Show & Tell

### Body

Without the "Block User" feature, he may be forced to leave the GitHub platform because users can't prevent some spamming, harassment and political attacks.

**Comment by LiteBrite82** [comment]
Hi @XuanChen1234 

Thanks for posting! 
Can you explain your post in more detail? I don't fully understand what you are communicating.

**Reply by XuanChen1234** [reply, marked as answer]
We're thinking about removing the "Block user" feature, we're probably going to do it again, there's a lot of options, but a lot of things are underway, so we'll wait to see what happens next.
![screenshot-1714455222254](https://github.com/community/community/assets/140480110/d3afeae2-611a-46e8-961e-741f9a9d52d4)
